docker/Dockerfile

```diff
@@ -391,6 +391,8 @@ COPY zalenium.sh \
  /home/seluser/
 COPY scm-source.json /
 
+RUN mkdir /home/seluser/videos
```

Comment:
Why do we need to create this directory when the image is built? This should be done implicitly when the volume is mounted.

Reply:
Because with this changes you might use a subfolder of videos for persistance.... So the folder is needed for the process not to fail.